css/css.md

```diff
@@ -76,6 +77,9 @@ The activity for the HTML, CSS, Bootstrap and jQuery lessons is a project that a
 
 [Click here to access the instructions for the CSS portion of the recipe-page project](https://github.com/Techtonica/curriculum/blob/master/projects/recipe-page/phase-2-css-prompt.md). Please don't move on to the prompts for the other topics yet.
 
+Check for [Inline CSS](https://www.codecademy.com/articles/html-inline-styles) contents while removing *style* tag from the webpage.
+
+Tutorials for changing and removing style tags are provided in the [CSS slides](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
```

Comment:
Can you add a ### Challenge section after this and ask them to remove a style tag from a real website themselves?  Give them 2 or 3 links to choose from where there's an obvious style tag in the head.

Reply:
I have added one as there are not many such websites. Hopefully, the content for CSS will help solve any challenge as it is very good and have been reviewed carefully..

Reply:
There are many websites... @pydevsg 
https://developers.google.com/
https://events.google.com/io/
And many more...

Reply:
I’m not saying find tutorials, although that link you found is a good resource. I’m saying find learners regular websites where they can remove an obvious style tag and see a difference. Then tell them to try it themselves in independent practice.

Reply:
These are great, but I had to remove linked stylesheets from amazon.in.  Can you rephrase this sentence to refer to style and link tags, and not mention inline?